### PR TITLE
docs: add reference to ossinsight

### PR DESCRIPTION
**Pull request by secustor**
## Changes
Add a section regarding contributions to the README and reference to our ossinsight page. https://ossinsight.io/analyze/renovatebot/renovate

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ Renovate is widely used in the developer community:
 
 ![Renovate Matrix](https://app.renovatebot.com/images/matrix.png)
 
+## Who contributes to Renovate?
+
+Renovate is build on a big community and actively invites contributions.
```
Comment (rarkins) [inline readme.md:49]:
```suggestion
Renovate is built on a big community and actively invites and supports contributions.
```

File: readme.md [hunk 2]
```diff
@@ -44,6 +44,11 @@ Renovate is widely used in the developer community:
 
 ![Renovate Matrix](https://app.renovatebot.com/images/matrix.png)
 
+## Who contributes to Renovate?
```
Comment (HonkingGoose) [inline readme.md:47]: We already have a section on contributing. I think we should move this part there. 😉 

> ## Contributing
>
> If you want to contribute to Renovate or get a local copy running, please read the instructions in [.github/contributing.md](.github/contributing.md).
> To get started look at the [list of good first issues](https://github.com/renovatebot/renovate/contribute).

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (rarkins) [reply readme.md:47]: The OSS Insight page isn't purely about contributions though

Comment (secustor) [reply readme.md:47]: So maybe this is broad enough?  🤔 
```suggestion
## Who is the Renovate community? 
```

Comment (rarkins) [reply readme.md:47]: How about "Renovate OSS Insights"?

**Review by rarkins — APPROVED** [review]

File: readme.md (hunk 2 above)
Comment (secustor) [reply readme.md]: Then it is decided ^^

**Pull request merged**